source/jormungandr/jormungandr/instance.py

```diff
@@ -162,10 +133,9 @@ def __init__(
         self.georef = georef.Kraken(self)
         self.planner = planner.Kraken(self)
 
-        street_network_configurations = _set_default_street_network_config(street_network_configurations)
-        self.street_network_services = street_network.StreetNetwork.get_street_network_services(
-            self, street_network_configurations
-        )
+        self.streetnetwork_backend_manager = StreetNetworkBackendManager(street_network_configurations)
```

Comment:
I would prefer having a manager shared between instances to be able to use the same instance of the backend class so circuit breaker, caches and others things can be shared.